openhands/runtime/client/runtime.py

```diff
@@ -146,9 +149,12 @@ def __init__(
         self.log_buffer: LogBuffer | None = None
 
         if self.config.sandbox.runtime_extra_deps:
-            logger.info(
+            logger.debug(
                 f'Installing extra user-provided dependencies in the runtime image: {self.config.sandbox.runtime_extra_deps}'
             )
+
+        self.event_stream = event_stream
```

Comment:
Just a little curious, before this PR, it was enough to do the super init call at line 180, which will initialize the event_stream. Are we using it before that now? If not, we shouldn't need this line.

Reply:
uhh, did I change that? 🤔

Reply:
Ohh, I think I had it moved up, because in earlier commits I was trying to use the event stream for the "send_status_message", back when we talked about getting to a session, maybe you remember 😂 
Looks like it's a remnant and I dont need it here anymore.